### PR TITLE
Chore version upgrades

### PR DESCRIPTION
**Pull request by mirestrepo**
Updates in this PR

- [x] Terraform
- [x] Ruby
- [x] Providers
- [x] Modules in GitHub Actions

This PR also changes from using a shared terraform container from `brownuniversity/kitchen-terraform` to building one from a local Dockerfile. I started that process for some of the modules last year, but this one didn't get that update.

### CODE REVIEW

**Review by singhsaluja — APPROVED** [review]
Thank you @mirestrepo

File: README.md [hunk 1]
```diff
@@ -37,49 +37,128 @@ code by adding a `module` configuration and setting its `source` parameter to UR
...
+### Merging Policy
+Use [GitLab Flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html#production-branch-with-gitlab-flow).
+
+* Create feature branches for features and fixes from default branch
+* Merge only from PR with review
+* After merging to default branch a release is drafted using a github action. Check the draft and publish if you and tests are happy
+
+### Version managers
+
+We recommend using [asdf](https://asdf-vm.com) to manage your versions of Terrafom and Ruby.
```
Comment (broestls) [inline README.md:99]: I thought this was just boilerplate language, but TIL about the asdf version manager. Neat!

**Review by broestls — APPROVED** [review]
LGTM! 🚀

File: README.md (hunk 1 above)
Comment (mirestrepo) [reply README.md:99]: Yeah, we've been using it more in our group. It's a nice concept, too many tools!. Hopefully the plugin ecosystem stay active and robust

**Pull request merged**